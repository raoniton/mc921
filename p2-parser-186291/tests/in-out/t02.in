class Main {
    public static void main(String[] args) {
        int n, r, sum = 0, temp;
    }
}