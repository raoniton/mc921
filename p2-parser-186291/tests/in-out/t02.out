Program:
    ClassDecl: ID(name=Main) @ 1:1
        MainMethodDecl: @ 2:5
            ID: args @ 2:38
            Compound: @ 2:44
                VarDecl: ID(name=n) @ 3:13
                    Type: int @ 3:9
                VarDecl: ID(name=r) @ 3:16
                    Type: int @ 3:9
                VarDecl: ID(name=sum) @ 3:19
                    Type: int @ 3:9
                    Constant: int, 0 @ 3:25
                VarDecl: ID(name=temp) @ 3:28
                    Type: int @ 3:9
